Documentation/Design/UserStory_WorldInteraction.htm

```diff
@@ -0,0 +1,86 @@
+<html>
+<head></head>
```

Comment:
I really like the idea of having a plain English description of features; as it is I look at the list of things that are supposed to be done the inner TDD guy in me cant help but think that actually none of them work, since i have yet to see tests that prove otherwise. ;)